docs/user/tutorial/70-page-management.rst

```diff
@@ -22,9 +25,40 @@ it's::
         how to find us
         people
 
-.. todo:: screenshots of following steps (if they are new steps)
+------------
 
-.. todo:: write up steps properly
+|page_tree_clean|
+
+    .. |page_tree_clean| image:: /user/tutorial/images/page_tree_clean.png
+       :alt: 'Page tree clean'
+       :width: 600
+
+
+To do this, please follow these steps:
+
+#. Add a new page by clicking on the **+ Add page** button. |add_page_from_tree|
+
+    .. |add_page_from_tree| image:: /user/tutorial/images/add_page_from_pagetree.png
+       :alt: 'Add Page from tree'
+       :width: 100
+
+#. Enter the title of the parent page.
+
+    .. image:: /user/tutorial/images/name_of_parent_page.png
+       :alt: Enter title of parent page
+       :width: 100%
+
+#. Hit **save** to create the page. |save_button|
+
+    .. |save_button| image:: /user/tutorial/images/save_button.png
+       :alt: 'Hit save button'
+       :width: 60px
```

Comment:
You don't need to specify the width of inline images such as buttons or icons - unless you have taken them on a Retina screen, in which case make them 50% of the actual size (in pixels).